backend/tests/test_api_integration.py

```
@@ -0,0 +1,437 @@
+    return {
+        "Authorization": f"Bearer {token}",
+        "Content-Type": "application/json"
+    }
+
+
+class TestHealthEndpoint:
+    """Test health check endpoint."""
+    
+    def test_health_check(self, auth_headers):
+        """Test that health endpoint returns 200 OK."""
+        response = requests.get(f"{SERVICE_URL}/api/health", headers=auth_headers)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)